docs/book/v3/psr6.md

```diff
@@ -138,3 +138,24 @@ returned as a value with exactly the same type.
 Not all adapters can natively store all these types. For instance, Redis stores booleans and integers as a string. Where
 this is the case *all* values will be automatically run through `serialize()` on save and `unserialize()` on get: you
 do not need to use a `Laminas\Cache\Storage\Plugin\Serializer` plugin.
+
+## Time-Related Operations
+
+By default, the PSR-6 cache decorator uses the systems timezone to determine cache expirations. This can be changed by
+passing an `lcobucci/clock` `Clock` implementation to the `CacheItemPoolDecorator#__construct`.
+By doing so, the method `CacheItemInterface#expiresAfter` uses that `Clock` to calculate the cache items TTL when passing
+a `DateInterval` instance.
```

Comment:
I'm not 100% sure we actually need a clock for this but there might be edge-cases where a TTL of for example `7 days` might get different results depending on DST transitions. Though I'm not really sure that makes that much of a difference for user-land. For the testing purposes it definitely makes sense as we can skip calling `sleep` as we can return different datetimes in different calls to `now` so we can "simulate" passing of a certain amount of time. 

```suggestion
By default, the PSR-6 cache decorator uses the systems time to determine cache expirations. This can be changed by
passing a [`stella-maris/clock`](https://packagist.org/packages/stella-maris/clock) `Clock` implementation to the `CacheItemPoolDecorator#__construct`.
By doing so, the method `CacheItemInterface#expiresAfter` uses that `Clock` to calculate the cache items TTL when passing
a `DateInterval` instance.
```